lib/jekyll-admin/path_helper.rb

```diff
@@ -52,14 +52,18 @@ def renamed?
 
     # Returns the path to the requested file's containing directory
     def directory_path
-      case namespace
-      when "collections"
-        sanitized_path File.join(collection.relative_directory, params["splat"].first)
-      when "data"
-        sanitized_path File.join(DataFile.data_dir, params["splat"].first)
-      else
-        sanitized_path params["splat"].first
-      end
+      sanitized_path(
+        case namespace
+        when "collections"
+          File.join(collection.relative_directory, params["splat"].first)
+        when "data"
+          File.join(DataFile.data_dir, params["splat"].first)
+        when "drafts"
+          File.join("_drafts", params["splat"].first)
+        else
+          params["splat"].first
+        end
+      )
```

Comment:
This could be further split and DRYed up as below:
```ruby

    # Returns the path to the requested file's containing directory
    def directory_path
      sanitized_path File.join(content_dir, params["splat"].first)
    end

    def content_dir
      case namespace
      when "collections"
        collection.relative_directory
      when "data"
        DataFile.data_dir
      when "drafts"
        "_drafts"
      else
        ""
      end
    end
```
Whatsay?

Reply:
LGTM 👍